src/features/commands/toxic.js

```diff
@@ -30,13 +42,23 @@ module.exports = {
                         { name: 'Probability', value: `**Toxicity:** ${toxicity.toxicity} \n **Insult:** ${toxicity.insult}` },
                         { name: 'Dotsimus combined probability', value: `${toxicity.combined}` }
                     );
-                interaction.editReply({
-                    type: 4,
-                    embeds: [embedResponse]
-                })
+                if(toxicity.toxicity > 0.80) {
```

Comment:
Lets lower it even more, as apparently certain racist word falls way beyond 0.80 😬
```suggestion
                if(toxicity.toxicity >= 0.60) {
```